code/game/objects/items/melee/misc.dm

```diff
@@ -85,6 +85,18 @@
 	icon_state = "sabresolgov"
 	item_state = "sabresolgov"
 
+/obj/item/melee/sabre/suns
+	name = "SUNS sabre"
+	desc = "A blade of solarian origin given to SUNS followers."
+	icon_state = "suns-sabre"
+	item_state = "suns-sabre"
+
+/obj/item/melee/sabre/suns/captain
+	name = "SUNS captain sabre"
+	desc = "An elegant blade awarded to SUNS captains and officers. Despite the higher craftmanship, it appears to be just as effective as a normal sabre."
```

Comment:
```suggestion
	desc = "An elegant blade awarded to SUNS captains and officers. Despite its higher craftmanship, it appears to be just as effective as a normal sabre."
```
Looking at the outfits, this blade is only awarded to captains, not officers. Maybe change the desc to reflect that, like...
```suggestion
	desc = "An elegant blade awarded to SUNS captains. Despite its higher craftmanship, it appears to be just as effective as a normal sabre."
```